### PR TITLE
Nicla Sense ME - Updated power consumptions [TC-299]

### PR DESCRIPTION
**Pull request by lunnea**
## What This PR Changes
- Updated PC

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/05.nicla/boards/nicla-sense-me/datasheet/datasheet.md [hunk 1]
```diff
@@ -253,6 +253,16 @@ All Arduino® boards have a built-in bootloader which allows flashing the board
...
 
+### Power Consumption
+| Description                                                     | Min  | Typ   | Max  | Unit |
+| ----------------------------------------------------------------| ---- | ----- | ---- | ---- |
+| Power consumption in standby                                    |      | 460   |      | uA   |
+| Power consumption with blink sketch                             |      | 960   |      | uA   |
+| Power consumption advertising with sensor poiling at 1Hz        |      | 2.5   |      | mA   |
+| Power consumption advertising with sensor poiling once per hour |      | 1.15  |      | mA   |
+
+**Note:** The measurments have been performed by activating temperature sensor, accelerometer and gyroscope, which have been configured with 1Hz sample rate and 1ms latency.
```
Comment (jhansson-ard) [inline content/hardware/05.nicla/boards/nicla-sense-me/datasheet/datasheet.md:264]:
```suggestion
**Note:** The measurements have been performed by activating temperature sensor, accelerometer and gyroscope, which have been configured with 1Hz sample rate and 1ms latency.
```

File: content/hardware/05.nicla/boards/nicla-sense-me/datasheet/datasheet.md [hunk 2]
```diff
@@ -253,6 +253,16 @@ All Arduino® boards have a built-in bootloader which allows flashing the board
 ## Mechanical Information
 ![](assets/niclaSenseMEMech.png)
 
+### Power Consumption
+| Description                                                     | Min  | Typ   | Max  | Unit |
+| ----------------------------------------------------------------| ---- | ----- | ---- | ---- |
+| Power consumption in standby                                    |      | 460   |      | uA   |
+| Power consumption with blink sketch                             |      | 960   |      | uA   |
+| Power consumption advertising with sensor poiling at 1Hz        |      | 2.5   |      | mA   |
```
Comment (martab1994) [inline content/hardware/05.nicla/boards/nicla-sense-me/datasheet/datasheet.md:261]: Power consumption advertising with sensor polling at 1Hz

File: content/hardware/05.nicla/boards/nicla-sense-me/datasheet/datasheet.md [hunk 3]
```diff
@@ -253,6 +253,16 @@ All Arduino® boards have a built-in bootloader which allows flashing the board
...
+| Power consumption advertising with sensor poiling once per hour |      | 1.15  |      | mA   |
```
Comment (martab1994) [inline content/hardware/05.nicla/boards/nicla-sense-me/datasheet/datasheet.md:262]: Power consumption advertising with sensor polling once per hour

**Review by martab1994 — CHANGES_REQUESTED** [review]
Just a couple of typos. Thanks!!

**Review by martab1994 — APPROVED** [review]

**Pull request merged**